Gemfile.lock

```diff
@@ -34,43 +34,43 @@ GEM
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.2)
+    i18n (1.8.3)
       concurrent-ruby (~> 1.0)
     json (2.3.0)
-    jwt (2.1.0)
-    jwt_signed_request (2.5.1)
-      jwt (>= 1.5.0, < 2.2.0)
+    jwt (2.2.1)
+    jwt_signed_request (2.5.4)
+      jwt (>= 1.5.0)
       rack
-    loofah (2.4.0)
+    loofah (2.6.0)
```

Comment:
### [loofah](https://github.com/flavorjones/loofah)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.4.0 → 2.6.0

[_[change-log](https://github.com/flavorjones/loofah/blob/master/CHANGELOG.md), [source-code](https://github.com/flavorjones/loofah)_]





<details>
<summary>Commits</summary>

A change of **34** commits. See the full changes on [the compare page](https://github.com/flavorjones/loofah/compare/v2.4.0...v2.6.0).

These are the first 10 commits:
- (d0b807e) [Add project metadata to the gemspec](https://github.com/flavorjones/loofah/commit/d0b807ea4955176b341aadef45feec8fc51a08f4)
- (1c0b7c6) [Update mailing list URI to Google Groups](https://github.com/flavorjones/loofah/commit/1c0b7c6d429a7f8e1fd3204b08fbcf86999a7f98)
- (199be72) [bump dev gemspec](https://github.com/flavorjones/loofah/commit/199be72528b0f8440e2394cdaf90efe2d8fb98c6)
- (ad7d598) [run `rufo` on all lib files](https://github.com/flavorjones/loofah/commit/ad7d5986bc1f74a4527955ecd5f124de2ea54893)
- (ed153a7) [run `rufo` on test files](https://github.com/flavorjones/loofah/commit/ed153a71cb5ce1c16aded36b5f16875fdac83dfe)
- (fc577c4) [Merge pull request #170 from orien/gem-metadata](https://github.com/flavorjones/loofah/commit/fc577c4dc66f9349edd51d1b1edc1a4c83fd4eec)
- (5be3eee) [CSS sanitizer allows "ch" length unit](https://github.com/flavorjones/loofah/commit/5be3eee454f60d00791690e588489b1dd216068d)
- (7c025e0) [CSS sanitizer allows vw length unit](https://github.com/flavorjones/loofah/commit/7c025e0eb45b846f7f673f61c275b53066f617ce)
- (e3fd485) [CSS sanitizer allows vh length unit](https://github.com/flavorjones/loofah/commit/e3fd485a4d02afae00e4ffd86fee89b635530be0)
- (bd97be9) [CSS sanitizer allows Q length unit](https://github.com/flavorjones/loofah/commit/bd97be9a92b58a4eb37796a038cc9115c80b1c81)


</details>